File: CS/T167275/Reports/SampleReport.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;
using DevExpress.XtraPrinting;

namespace T167275.Reports {
    public partial class SampleReport : DevExpress.XtraReports.UI.XtraReport {
        public SampleReport(IBasePrintable gridViewPrintable, IBasePrintable pivotGridPrintable) {
            InitializeComponent();
            pcGridView.PrintableComponent = gridViewPrintable;
            pcPivotGrid.PrintableComponent = pivotGridPrintable;
        }

    }
}

File: VB/T167275/Reports/SampleReport.vb
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI
Imports DevExpress.XtraPrinting

Namespace T167275.Reports
	Partial Public Class SampleReport
		Inherits DevExpress.XtraReports.UI.XtraReport

		Public Sub New(ByVal gridViewPrintable As IPrintable, ByVal pivotGridPrintable As IPrintable)
			InitializeComponent()
			pcGridView.PrintableComponent = gridViewPrintable
			pcPivotGrid.PrintableComponent = pivotGridPrintable
		End Sub

	End Class
End Namespace
